### DISCUSSION TITLE
0.27.0

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#368] [Announcements]
Introduces changes to the interface for custom Javascript components. This now allows JS modules to export a ``mount(element, component, props)`` function which can be used to bind new elements to the DOM instead of using the application's own React instance and specifying React as a peer dependency. This avoids a wide variety of potential issues with implementing custom components and opens up the possiblity for a wider variety of component implementations.

- modules with mount func should not have children - 94d006c
- limit to flask<2.0 - e7c11d0
- federate modules with mount function - bf63a62

<hr /><em>This discussion was created from the release <a href='https://github.com/idom-team/idom/releases/tag/0.27.0'>0.27.0 - Module Mount Function</a>.</em>

**Comment by Archmonger** [comment]
Definitely interested in hearing more about this implementation!
